webpack.mix.js

```diff
@@ -0,0 +1,15 @@
+const { mix } = require('laravel-mix');
```

Comment:
No, I didn't do a refactor from `elixir` to `mix` as a part of this, I figured that could be done as a separate effort since it doesn't really effect upgrading Laravel itself. But Shift still added this file. Would you prefer it just be removed until we refactor mix?